### ISSUE TITLE
Interpreting Question and Answering with Bert Part 1 - issue in predict script

### ISSUE THREAD

**Issue by IlanaL1** [pytorch/captum#683]
## 📚 Documentation

Interpreting Question and Answering with Bert Part 1, there is an error in the predict function. The function currently returns a string for start and end positions. 

I believe the correct function definition should be something like in Interpreting Question and Answering with Bert Part II, 

def predict(inputs, token_type_ids=None, position_ids=None, attention_mask=None):
    output = model(inputs, token_type_ids=token_type_ids,
                 position_ids=position_ids, attention_mask=attention_mask, )
    return output.start_logits, output.end_logits, output.attentions

squad_pos_forward_func may also need updating.

**Comment by IlanaL1** [comment]
Alternatively, the cell that calls this function further down in the tutorial may call it incorrectly
```
start_scores, end_scores = predict(input_ids, \
                                   token_type_ids=token_type_ids, \
                                   position_ids=position_ids, \
                                   attention_mask=attention_mask)
```

**Comment by craymichael** [comment]
Hi, I was unable to reproduce the issue. Since this issue is old, I will keep it closed for now. But if you're still having it, please reopen and sorry for the delay.

**Comment by craymichael** [comment]
Checked again against the Captum v0.9.0 tutorial source. Minimal check:

```python
from pathlib import Path
nb = Path("tutorials/Bert_SQUAD_Interpret.ipynb").read_text()
assert "return output.start_logits, output.end_logits" in nb
```

Behavior: assertion passes with no errors; the Part 1 `predict` helper returns tensor logits rather than strings.
